cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -314,11 +307,20 @@ if(@HUNTER_STATUS_DEBUG@)
 endif()
 
 if(CMAKE_HOST_WIN32)
-  set(bootstrap_cmd "bootstrap.bat")
-  set(b2_cmd "b2")
+  if("@MSVC@")
+    file(WRITE "@HUNTER_PACKAGE_DOWNLOAD_DIR@/patched_boostrap.bat" "\
```

Comment:
I think this will look better if you put this code into template file and call only `configure_file` here. Also it will fits DRY better since we need to use the same code in `url_sha1_boost.cmake.in`.

How it's implemented in Qt:
* https://github.com/ruslo/hunter/blob/b2b9cca56c9c5d993c9273b39c0c970cffcf1eb5/cmake/projects/Qt/schemes/url_sha1_qt.cmake.in#L402-L406
* https://github.com/ruslo/hunter/blob/b2b9cca56c9c5d993c9273b39c0c970cffcf1eb5/cmake/projects/Qt/schemes/url_sha1_qt.cmake.in#L444-L450

Reply:
Please resolve this.